build.sbt

```diff
@@ -4,4 +4,9 @@ version := "0.1"
 
 scalaVersion := "2.13.5"
 
-libraryDependencies += "org.scalatest" %% "scalatest" % "3.2.5" % "test"
\ No newline at end of file
+libraryDependencies += "org.scalatest" %% "scalatest" % "3.2.5" % "test"
+libraryDependencies += "dev.zio" %% "zio" % "1.0.8"

```

Comment:
Nothing major, but consistency, either add all as 
```scala
libraryDependencies += ...
libraryDependencies += ...
````

or 
```scala
libraryDependencies ++= Seq(
... , 
...
)
```
I prefer the second way